docs/proposals/shipwright-website.md

```diff
@@ -0,0 +1,125 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: shipwright-website
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@otaviof"
+  - "@SaschaSchwarze0"
+approvers:
+  - "@qu1queee"
+  - "@sbose78"
+creation-date: 2020-09-14
+last-updated: 2020-09-14
+status: implementable
+---
+
+# Shipwright Website
+
+## Release Signoff Checklist
+
+- [ x ] Enhancement is `implementable`
```

Comment:
Change these two to `[x]` so that they can show correctly.

Reply:
fixed in update